### PR TITLE
Added routines to optimize OODdaq offset calculation

### PR DESCRIPTION
**Pull request by MichaelHuth**
I cleaned the code up and corrected a bug where it aborted in some cases the offset calculation too early.
That resulted in a too low offset.

I updated and checked the oodDAQ tests.

The oodDAQ resolution was removed, it is always the one sample point now (in wavebuilder scale)

### CODE REVIEW

File: Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf [hunk 1]
```diff
@@ -6,6 +6,245 @@
...
+static Function OOD_GetThresholdLevel(stimSet)
+	WAVE stimSet
+
+	variable minVal = WaveMin(stimset)
+	return minVal + (WaveMax(stimset) - minVal) * 0.10
+End
+
+/// @brief retrieves regions with signal from a 1D data wave, used for stimsets
+/// @param[in] stimset 1D wave containing stimset data
+/// @param[in] prePoints pre delay in points the regions get expanded at the rising edge
```
Comment (timjarsky) [inline Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf]: @MichaelHuth minor comment: how about:
 `prePoints oodDAQ pre delay in points.....` 
and 
 `postPoints oodDAQ post delay in points.....`

**Comment by MichaelHuth** [comment]
`OOD_Optimizer` is part of the old code, to run the new approach only please comment in line 738 in `OOD_GetResultWaves` the call to `OOD_CalculateOffsetsYoked(panelTitle, params)`.

**Comment by t-b** [comment]
@MichaelHuth 

Doxygen says:

```
Errors building the documentation
Doxygen says:
/home/thomasb/bamboo-agent-home/xml-data/build-dir/MIES-CG24-DOC/Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf:9: warning: argument 'stimset' of command @param is not found in the argument list of OOD_GetThresholdLevel(wave stimSet)
/home/thomasb/bamboo-agent-home/xml-data/build-dir/MIES-CG24-DOC/Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf:12: warning: The following parameters of OOD_GetThresholdLevel(wave stimSet) are not documented:
  parameter 'stimSet'
```

**Comment by MichaelHuth** [comment]
@t-b Could you test if yoking works?

**Comment by t-b** [comment]
@MichaelHuth I can't. Only @timjarsky can test yoking.

**Comment by t-b** [comment]
@MichaelHuth We can get rid of the "oodDAQ resolution" control. Use 0 for that value when writing it into the LBN.

**Comment by t-b** [comment]
@MichaelHuth Documentation still fails with

```
Errors building the documentation
Doxygen says:
/home/thomasb/bamboo-agent-home/xml-data/build-dir/MIES-CG24-DOC/Packages/MIES/MIES_OptimzedOverlapDistributedAcquisition.ipf:141: warning: The following parameters of OOD_GetFeatureRegions(WaveRefWave setRegions, wave offsets) are not documented:
  parameter 'offsets'
```

**Comment by t-b** [comment]
@MichaelHuth 
Functionality:
Works as expected! And is *much* faster.

Commitwise/Documentation:
Looks good.

I've added a fixup commit with two minor changes. One link in a comment and I also increased the version of the cache key, so that we can distinguish old and new algorithms. Please rebase if these are correct changes in your eyes.

**Comment by MichaelHuth** [comment]
@t-b the fixup is good, I missed the cache version increase and the comment reference to the now removed function.

**Pull request merged**